jenkinsapi/label.py

```diff
@@ -0,0 +1,42 @@
+"""
+Module for jenkinsapi labels
+"""
+from jenkinsapi.jenkinsbase import JenkinsBase
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class Label(JenkinsBase):
+
+    """
+    Class to hold information on labels that tied to a collection of jobs
+    """
+
+    def __init__(self, baseurl, labelname, jenkins_obj):
+        """
+        Init a label object by providing all relevant pointers to it
+        :param baseurl: basic url for querying information on a node
+        :param labelname: name of the label
+        :param jenkins_obj: ref to the jenkins obj
+        :return: Label obj
+        """
+        self.labelname = labelname
+        self.jenkins = jenkins_obj
+        self.baseurl = baseurl
+        JenkinsBase.__init__(self, baseurl)
+
+    def __str__(self):
+        return '%s' % (self.labelname)
+
+    def get_jenkins_obj(self):
+        return self.jenkins
+
+    def is_online(self):
+        return not self.poll(tree='offline')['offline']
+
+    def get_tied_jobs(self):
```

Comment:
Either this shall return list of ``Job`` objects, or it shall be named differently. Just to make API a bit helpful: if something returns jobs, it shall return a list of objects and not just names.

I'd have it as ``tied_jobs()`` return a list of Jobs and ``tied_job_names`` returning job names.